openverse_catalog/dags/providers/provider_workflows.py

```diff
@@ -87,8 +87,8 @@ class ProviderWorkflow:
     load_timeout: timedelta = timedelta(hours=1)
     doc_md: str = ""
     media_types: Sequence[str] = ()
-    create_preingestion_tasks: Optional[callable] = None
-    create_postingestion_tasks: Optional[callable] = None
+    create_preingestion_tasks: Callable | None = None
+    create_postingestion_tasks: Callable | None = None
```

Comment:
Here, the previous code uses `callable` for type hinting instead of `typing.Callable` or `collections.abc.Callable`. I'm surprised flake8 didn't catch it.
This was the reason the PR was failing the tests.